src/lib/rnp.cpp

```diff
@@ -2711,7 +2712,7 @@ static void
 recipient_handle_from_pk_sesskey(rnp_recipient_handle_t  handle,
                                  const pgp_pk_sesskey_t &sesskey)
 {
-    memcpy(handle->keyid, sesskey.key_id, PGP_KEY_ID_SIZE);
+    memcpy(handle->keyid, sesskey.key_id.data(), sesskey.key_id.size());
```

Comment:
If this would be changed to PGP_KEY_ID_SIZE() then it would become unsafe if pgp_key_id_t would change to smaller value. So probably it better to just rely on fact that both are of the same size.

Assert is not encouraged to use, as far as I remember, but unable to fine that in development docs. @dewyatt do I correctly remember that we should not use asserts?

Reply:
I don't think we have an official guideline on assert, but IMO it should be used sparingly, assertions with side effects should be avoided, etc.

In this case, a static assert might be good:
```cpp
static_assert(sizeof(handle->keyid) == sizeof(sesskey.key_id));
```

Reply:
@dewyatt thanks, `static_assert()` looks good.